spirv_reflect.c

```diff
@@ -71,7 +71,8 @@ enum {
 
 // clang-format off
 enum {
-  MAX_NODE_NAME_LENGTH      = 1024,
+  MAX_NODE_NAME_LENGTH        = 1024,
+  MAX_RECURSIVE_POINTER_CHECK = 128
```

Comment:
nit: missing comma at end

Can you also add a one line comment on what this is used for?